python/pyarrow/lib.pyx

```diff
@@ -37,6 +37,10 @@ arrow_init_numpy()
 # (used from some of our C++ code, see e.g. ARROW-5260)
 import_pyarrow()
 
+cdef libarrow.ArrowToPython ARROW_TO_PYTHON
```

Comment:
This seems rather cryptic. What does this do? Is this a globally defined converter?
(is initialization costly?)

Reply:
yes, globally defined converter.  This will likely change based on the outcome of simplying "as_py" but before it required getting references to Pandas types.  In the long run we likely need to import a few others (timestamp, timedelta, decimal, etc).  It seems like for Pandas we used C++ static variables.  It felt cleaner to me make it an explicit object.  Again don't feel too strongly here and if we eliminate the immediate need to share Pandas conversion logic this could go away.  Do you have a preference on the pattern here?

Reply:
I'm ok with having a wrapper object, however we shouldn't import Pandas unconditionally at startup as that's quite costly.

Reply:
It no longer needs to do any imports for this PR (pandas objects are left in pandas related classes).  It was doing the import lazily on first method calls.